README.md

```diff
@@ -21,6 +21,32 @@ A developer and [contribution guide](./CONTRIBUTING.md) is also available.
 
 Please refer to the support page [here](./docs/modules/ROOT/pages/support.adoc).
 
+## Local build
+
+First, checkout the code using repo:
+
+```
+# Get code
+$ mkdir cmos
+$ cd cmos
+$ repo init -u https://github.com/couchbase/manifest -m couchbase-observability-stack/master.xml
+$ repo sync
+$ cd couchbase-observability-stack
```

Comment:
These steps will need to be different for someone who isn't in Couchbase - they can just `git clone` like normal, though they will need to run `make container -e OSS=true` rather than just bare `make container`

Reply:
Added a new section to address this